File: Java/AutomationFramework/src/com/app/UITests/LoginPageTests/LoginLogoTest.java
package com.app.UITests.LoginPageTests;

import com.app.pages.LoginPage;
import org.testng.Assert;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class LoginLogoTest extends BaseLoginTest {
    LoginPage Login_Page;

    @BeforeMethod
    public void SetUp() {
        Login_Page = LoginPage.GetLoginPage()
                .Init_Elements();
    }

    @Test
    public void LogoTest()
    {
        boolean isLogoPresent = Login_Page.IsLogoPresent();
        Assert.assertEquals(isLogoPresent, true);
    }

}

File: DotNet/Framework/AutomationFramework/Tests.UITests/LoginPageTests/LoginLogoTest.cs
using NUnit.Framework;
using Pages;

namespace Tests.UITests.LoginPageTests
{
    [Parallelizable]
    [TestFixture]
    public class LoginLogoTest : BaseTestFixtures
    {
        LoginPage Login_Page;

        [SetUp] 
        public void Setup()
        {
            Login_Page = LoginPage.Login_Page;
        }

        [Test]
        public void LogoTest() 
        {
            bool isLogoPresent = false;
            Login_Page.IsLogoPresent(out isLogoPresent);

            Assert.That(isLogoPresent, Is.EqualTo(true));
        }
    }
}
